src/components/pages/Home/HowItWorks.tsx

```diff
@@ -0,0 +1,244 @@
+import { FC, useCallback, useState } from "react";
+
+enum Section {
+ PlantSeeds = "PLANT_SEEDS",
+ BuildTree = "BUILD_TREE",
+ IdentifyRelevantTheories = "IDENTIFY_RELEVANT_THEORIES",
+ ExtractYourLiteratureList = "EXTRACT_YOUR_LITERATURE_LIST",
+}
+
+const HowItWorks: FC = () => {
+ const [show, setShow] = useState<Section>(Section.PlantSeeds);
+
+ const toggleShow = useCallback((label: Section) => {
+  setShow((curr) => {
+   if (curr === label) {
+    return curr;
+   }
+   return label;
+  });
+ }, []);
+
+ return (
+  <section
+   id="how-it-works"
+   aria-label="Features for running your books"
```

Comment:
Let's change this aria labe to `How Tree of Science works`.

Reply:
I have already done the change :dog: